content/en/how-to-guides/configuration-management/change_management.adoc

```diff
@@ -0,0 +1,60 @@
+{{% pageinfo %}}
+These guidelines assume that you're familiar with the GitHub workflow.
+{{% /pageinfo %}}
+
+== How to use the kanban board ==
+
+. Assign yourself to a new or existing card or issue in the https://github.com/orgs/tetrabiodistributed/projects/1[Project Tetra - kanban board]. Consult the GitHub project admin for access, if needed.
+  * Documentation related issues should be created in the https://github.com/tetrabiodistributed/project-tetra-docs/issues[Project Tetra docs issues] and added to the https://github.com/orgs/tetrabiodistributed/projects/1[Project Tetra - kanban board].
+
+  * Source code related issues should be created in the https://github.com/tetrabiodistributed/project-tetra/issues[Project Tetra issues] and added to the https://github.com/orgs/tetrabiodistributed/projects/1[Project Tetra - kanban board].
+
+  * Non-documentation or source code related issues should be added as a card to the https://github.com/orgs/tetrabiodistributed/projects/1[Project Tetra - kanban board].
+
+. Record relevant discussions in the comment section of the GitHub issues.
+
+== How to make documentation changes ==
+
+. Create a new branch, or checkout an existing branch, in the https://github.com/helpfulengineering/project-tetra-docs[Project Tetra docs repo]. The branch name should describe the changes to be made, followed by its associated issue number, e.g., `add-tricoder-with-touchscreen-docs-1701`. If no such issue exists, <<_creating_an_issue, create a new issue>>.
+
+. Write or update the appropriate documentation page relevant to the design changes to be made in your working branch.  All documentation should be nested under link:../../tutorials[Tutorials], link:../../how-to-guides[How-To Guides], link:../../explanations[Explanations], and/or link:../../references[References]. (See <<#_how_to_write_good_documentation, How to write good documentation>>)
+
+. Commit the documentation changes to your working branch in the https://github.com/tetrabiodistributed/project-tetra-docs[Project Tetra docs repo].
+
+. Create a pull request from your working branches in the https://github.com/tetrabiodistributed/project-tetra-docs[Project Tetra docs repo] and https://github.com/helpfulengineering/project-tetra[Project Tetra repo] to be merged into their respective main branch.
+
+== How to make design changes ==
+
+Design changes include changes to CAD, STL, Drawings, or software.
+
+. Update the documentation to reflect the design changes to be made. (See <<_how_to_make_documentation_changes, How to make documentation changes>>)
+
+. Create a new working branch, or checkout an existing branch, in the https://github.com/helpfulengineering/project-tetra[Project Tetra repo]. The branch name should describe the change to be made, followed by its associated issue number, e.g., `add-tricoder-with-touchscreen-1031`. If no such issue exists, create a new issue in the https://github.com/tetrabiodistributed/project-tetra/issues[Project Tetra issues].
```

Comment:
From a hardware standpoint, it will be tedious and difficult to follow this process. Since each part file that gets uploaded with be reviewed beforehand, it would be better to allow for bulk changes to be made. Perhaps creating a new issue for linking the PR that would also tag the individual issues being addressed would be a decent way to handle this? I'm open to suggestions, but I don't know that anyone on the hardware side will follow this...If this is ONLY relevant for changing hardware documentation, then I think it's fine.

Reply:
This section actually refers to non-documentation changes, so you bring up an important concern. Would it be helpful if the verbiage was extended to mention that a Pull Request can close multiple issues and that the branch name could be named after an _umbrella_ issue that would references constituent issues to be closed?